app/logic/Calendar/IncomingActions.ts

```diff
@@ -0,0 +1,64 @@
+import type { Calendar } from "./Calendar";
+import type { Event } from "./Event";
+import { Scheduling, ResponseType, type Responses } from "./Invitation";
+import type { EMail } from "../Mail/EMail";
+import { assert } from "../util/util";
+
+export class IncomingActions {
```

Comment:
Please rename to `IncomingInvitation` and move to app/logic/Calendar/Invitation/IncomingInvitation.ts . Ditto for `ActiveSyncIncomingInvitation` etc.